static/js/navigation.js

```diff
@@ -13,6 +14,10 @@ navDropdowns.forEach(function(dropdown) {
       dropdownWindow.classList.remove('fade-animation');
     }
 
+    if (dropdownWindowOverlay.classList.contains('fade-animation')) {
+      dropdownWindowOverlay.classList.remove('fade-animation');
```

Comment:
Won't remove just do nothing if it doesn't contain it? Saves the conditional.

Reply:
Good point. Fixed